docs/queue.rst

```diff
@@ -53,9 +53,9 @@ you can run:
  - ``cleanup_mail`` will delete mails created before an X number of days
    (defaults to 90).
 
- - ``status_mail`` the intent of this commant is to allow systems as nagios to
-   be able to ask the queue about its status. It returns as string with than
-   can be parses as ``(?P<queued>\d+)/(?P<deferred>\d+)/(?P<seconds>\d+)``
+ - ``status_mail`` returns a output for allow to monitoring some parameters.
+   By default ``queued/deferred/second``. If you prefere the json format,
+   please, add the ``--json-format`` parameter when call the command.
```

Comment:
The previous description and the new one you have proposed have some errors. I think the following description would be clearer:

``status_mail`` command allows systems like Nagios to able to check the queue status.
It returns a string that can be parsed as ``(?P<queued>\d+)/(?P<deferred>\d+)/(?P<seconds>\d+)``.
Adding ``--json-format`` option, the output will be in JSON format.

Reply:
Done!